### PR TITLE
Add menu item classnames

### PR DESCRIPTION
**Pull request by spefley**
Add two new props to menu item to allow you to apply a classname to the text and label wrapper elements

### CODE REVIEW

**Comment by palantirtech** [comment]
Thanks for your interest in [palantir/blueprint](https://github.com/palantir/blueprint), @spefley! Before we can accept your pull request, you need to sign our contributor license agreement - just visit https://cla.palantir.com/ and follow the instructions. Once you sign, I'll automatically update this pull request.

File: packages/core/src/components/menu/menuItem.tsx [hunk 1]
```diff
@@ -21,6 +21,9 @@ export interface IMenuItemProps extends IActionProps, ILinkProps {
     /** Item text, required for usability. */
     text: React.ReactNode;
 
+    /** Classname for text wrapper element. */
```
Comment (giladgray) [inline packages/core/src/components/menu/menuItem.tsx]: match docs for `className`.

>     /** A space-delimited list of class names to pass along to the element that wraps `text`. */

File: packages/core/src/components/menu/menuItem.tsx [hunk 2]
```diff
@@ -51,6 +54,11 @@ export interface IMenuItemProps extends IActionProps, ILinkProps {
      */
     labelElement?: React.ReactNode;
 
+    /**
+     * Classname for the right-aligned label wrapper element.
```
Comment (giladgray) [inline packages/core/src/components/menu/menuItem.tsx]: same

File: packages/core/src/components/menu/menuItem.tsx (hunk 1 above)
Comment (giladgray) [reply packages/core/src/components/menu/menuItem.tsx]: alphabetize this prop please.

**Pull request merged**